packages/front-end/services/UserContext.tsx

```diff
@@ -351,13 +375,12 @@ export function UserContextProvider({ children }: { children: ReactNode }) {
         },
         settings: currentOrg?.organization?.settings || {},
         license: data?.license,
-        // @ts-expect-error TS(2322) If you come across this, please fix it!: Type 'SSOConnectionInterface | null | undefined' i... Remove this comment to see the full error message
-        enterpriseSSO: currentOrg?.enterpriseSSO,
+        enterpriseSSO: currentOrg?.enterpriseSSO || undefined,
         accountPlan: currentOrg?.accountPlan,
         commercialFeatures: currentOrg?.commercialFeatures || [],
         apiKeys: currentOrg?.apiKeys || [],
-        // @ts-expect-error TS(2322) If you come across this, please fix it!: Type 'OrganizationInterface | undefined' is not as... Remove this comment to see the full error message
-        organization: currentOrg?.organization,
+        organization: currentOrg?.organization || {},
```

Comment:
I think this will break some things.  There are some places in the code that check `if (userContext.organization)`

Reply:
To get rid of the ts-expects-error warning, we should probably change the type of UserContextValue so that `organization` can be undefined

Reply:
Changing it so it can be undefined just triggered some type errors in 17 files. Will leave the `ts-expect-error` and fix in a fast-follow to unblock